Projecten3/EvaAndroid/app/src/main/java/projecten3_h6/evaandroid/MainActivity.java

```diff
@@ -38,9 +51,31 @@ protected void onCreate(Bundle savedInstanceState) {
         drawer.setDrawerListener(toggle);
         toggle.syncState();
 
+
+        app = (EvaApplication)getApplicationContext();
+        //getting the user from the saved file
+        app.setUser(getUserOutOfFile());
+
         displaySelectedScreen(R.id.nav_progress);
 
         navigationView.setNavigationItemSelectedListener(this);
+
+    }
+
+    private User getUserOutOfFile(){
```

Comment:
Don't forget to create a new User with createNewUser() when there is no file to get a user from.